tools/nodejs/lib/duckdb.js

```diff
@@ -305,16 +310,20 @@ var default_connection = function (o) {
 
 
 /**
+ * Closes database instance
  * @method
  * @param callback
  * @return {void}
  */
-Database.prototype.close;
+Database.prototype.close = function() {
+    this.default_connection = null
+    this.close_internal();
```

Comment:
I believe this needs to call the input callback, so the caller knows when the close is done (same API that other functions use).

Specifically, I was able to get this to work using
```suggestion
     this.close_internal(arguments['0']);
```

Reply:
Indeed that was an oversight. I added your suggestion (in a slightly different way).

Reply:
lgtm!